src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -367,181 +364,213 @@ private void testQuestionTypesSubmitAction(){
     }
     
-    private void testMsqSubmitAction(){
+    private void testMsqSubmitAction() {
         ______TS("test submit actions for msq.");
         
-        //Test input disabled
+        // Test input disabled
         int qnNumber = 3;
         int responseNumber = 0;
+        
         assertTrue(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         
-        //TODO: test that the recipient selection is also disabled
+        // Test that the recipient selection is disabled and not visible
+        assertFalse(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
+        assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
     }
     
-    private void testNumScaleSubmitAction(){
+    private void testNumScaleSubmitAction() {
         ______TS("test submit actions for numscale questions.");
         
-        //Test input disabled
+        // Test input disabled
         int qnNumber = 4;
         int responseNumber = 0;
+        
         assertTrue(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
-
-        //TODO: test that the recipient selection is also disabled
         
-        //Test input entered are valid numbers for the question.
+        // Test that the recipient selection is disabled and not visible
+        assertFalse(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
+        assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
+        
+        // Test input entered are valid numbers for the question.
         submitPage = loginToInstructorFeedbackSubmitPage("IFSubmitUiT.instr", "Open Session");
+        
         qnNumber = 14;
         responseNumber = 0;
         
-        submitPage.fillResponseTextBox(14, 0, "2.5");
-        assertEquals("2.5",submitPage.getResponseTextBoxValue(14, 0));
-        
-        submitPage.fillResponseTextBox(14, 0, "ABCD");
-        assertEquals("",submitPage.getResponseTextBoxValue(14, 0));
-        
-        fillResponseTextBoxWithRecheck(14, 0, "0", "1");
-        
-        submitPage.fillResponseTextBox(14, 0, "-1");
-        assertEquals("1",submitPage.getResponseTextBoxValue(14, 0));
-        
-        fillResponseTextBoxWithRecheck(14, 0, "6", "5");
```

Comment:
It was used here and at line 432, it's kinda out of place because some tests are tested one way and some tests here are tested another way, I believe it was meant for something else but it has since been removed but someone saw the method and used it here.